pygmt/helpers/testing.py

```diff
@@ -156,6 +156,8 @@ def download_test_data():
     datasets = [
         # Earth relief grids
         "@earth_gebco_01d_g",
+        "@earth_gebcosi_01d_g",
+        "earth_gebcosi_15m_p",
```

Comment:
```suggestion
        "@earth_gebcosi_15m_p",
```

Reply:
Keep forgetting the `@` at the start; thanks!